README.md

```diff
@@ -195,22 +192,21 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 
 ### Playbooks
 
-* [Demisto Playbooks Collection](https://www.demisto.com/category/playbooks/) - Playbooks collection.
 * [IRM](https://github.com/certsocietegenerale/IRM) - Incident Response Methodologies by CERT Societe Generale.
 * [IR Workflow Gallery](https://www.incidentresponse.com/playbooks/) - Different generic incident response workflows, e.g. for malware outbreak, data theft, unauthorized access,... Every workflow constists of seven steps: prepare, detect, analyze, contain, eradicate, recover, post-incident handling. The workflows are online available or for download.
 * [PagerDuty Incident Response Documentation](https://response.pagerduty.com/) - Documents that describe parts of the PagerDuty Incident Response process. It provides information not only on preparing for an incident, but also what to do during and after. Source is available on [GitHub](https://github.com/PagerDuty/incident-response-docs).
 
 ### Process Dump Tools
 
-* [Microsoft User Mode Process Dumper](http://www.microsoft.com/en-us/download/details.aspx?id=4060) - Dumps any running Win32 processes memory image on the fly.
+* [Microsoft ProcDump](https://docs.microsoft.com/en-us/sysinternals/downloads/procdump) - Dumps any running Win32 processes memory image on the fly.
 * [PMDump](http://www.ntsecurity.nu/toolbox/pmdump/) - Tool that lets you dump the memory contents of a process to a file without stopping the process.
 
 ### Sandboxing/reversing tools
 
 * [AMAaaS](https://amaaas.com/index.php/AMAaaS/dashboard) - Android Malware Analysis as a Service, executed in a native Android environment.
 * [Any Run](https://app.any.run/) - Interactive online malware analysis service for dynamic and static research of most types of threats using any environment.
-* [CAPE](https://github.com/ctxis/CAPE) - Malware Configuration And Payload Extraction.
-* [Cuckoo](https://github.com/cuckoobox) - Open Source Highly configurable sandboxing tool.
+* [CAPEv2](https://github.com/kevoreilly/CAPEv2) - Malware Configuration And Payload Extraction.
+* [Cuckoo](https://github.com/cuckoosandbox) - Open Source Highly configurable sandboxing tool.
```

Comment:
why not link https://github.com/cuckoosandbox/cuckoo here ?

Reply:
I used the project link since this is what was used before as well. But I think it is better to use the direct link to the sandbox itself.